src/MyEventEmitter.js

```diff
@@ -1,12 +1,87 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, callBack) {
```

Comment:
For better readability, use 'this.events[eventName] = this.events[eventName] || [];', instead of using an 'if' statement to initialize the array.